docs/guides/create-js-plugin.md

```diff
@@ -50,34 +50,30 @@ Let's ensure all of your project's dependencies are installed. From inside your
 Your project should look something like this:
 
 ```
-schema.graphql                # Schema
+web3api.plugin.yaml           # Plugin Manifest
 src/
```

Comment:
common/root-level index.ts is missing and we should also show and explain codegen file structure

Reply:
okay makes sense, will add

Reply:
Hmmm actually I'm thinking we leave the codegen files out of this guide, since I think it's a bit too much information if someone is just trying to get started.